code/game/objects/items/explosives/mine.dm

```diff
@@ -82,29 +121,45 @@
 
 	if(check_for_obstacles(user))
 		return
-
 	user.visible_message(SPAN_NOTICE("[user] finishes deploying [src]."), \
 		SPAN_NOTICE("You finish deploying [src]."))
-
+	if(needs_digging)
+		playsound(loc, 'sound/weapons/flipblade.ogg', 25, 1)
+		user.visible_message(SPAN_NOTICE("[user] pulls the pin on \the [src]."), \
+			SPAN_NOTICE("You pull the activation pin and prepare it to be buried."))
+		user.drop_inv_item_on_ground(src)
+		anchored = TRUE
+		return
 	deploy_mine(user)
 
 /obj/item/explosive/mine/proc/deploy_mine(mob/user)
 	if(!hard_iff_lock && user)
 		iff_signal = user.faction
-
 	cause_data = create_cause_data(initial(name), user)
 	anchored = TRUE
 	playsound(loc, 'sound/weapons/mine_armed.ogg', 25, 1)
 	if(user)
 		user.drop_inv_item_on_ground(src)
-	setDir(user ? user.dir : dir) //The direction it is planted in is the direction the user faces at that time
+		setDir(user ? user.dir : dir) //The direction it is planted in is the direction the user faces at that time
 	activate_sensors()
 	update_icon()
 
 
 //Disarming
-/obj/item/explosive/mine/attackby(obj/item/W, mob/user)
-	if(HAS_TRAIT(W, TRAIT_TOOL_MULTITOOL))
+/obj/item/explosive/mine/attackby(obj/item/thing, mob/user)
+	if(needs_digging && HAS_TRAIT(thing,  TRAIT_TOOL_SHOVEL) && !active && anchored)
+		user.visible_message(SPAN_NOTICE("[user] starts burying \the [src]."), \
+			SPAN_NOTICE("You start burying \the [src]."))
+		playsound(user.loc, 'sound/effects/thud.ogg', 40, 1, 6)
+		if(!do_after(user, 30, INTERRUPT_NO_NEEDHAND, BUSY_ICON_BUILD))
+			user.visible_message(SPAN_WARNING("[user] stops burying \the [src]."), \
+			SPAN_WARNING("You stop burying [src]."))
```

Comment:
```suggestion
			SPAN_WARNING("You stop burying [src]."))
			return
```

Reply:
You didn't do this review. Don't wordlessly resolve reviews that you don't do.

Reply:
apologies, this was muddled like the other one